app/views/shared/_message_list_contact_record.html.erb

```diff
@@ -3,60 +3,54 @@
   to add new messages to the page as a result of a websocket broadcast
 !-->
 <% self.extend(ContactRecordHelper) %>
-<li class="message message--<%= contact_record.contact_record_type %>" id="<%= "last-item" if local_assigns[:is_last_item] %>">
-  <div class="message__heading">
-    <div>
-      <span class="message__author">
-        <%= contact_record.try(:author) %>
-      </span>
+<li class="message message--<%= contact_record.contact_record_type %>" id=<%= "last-item" if local_assigns[:is_last_item] %>>
+  <div class="message__content">
+    <div class="message__heading">
+      <div class="message__authority">
```

Comment:
love this new wrapper class. "message__attribution" feels a little more intuitive to me.